### DISCUSSION TITLE
PyCharm 2022.1.1 (Professional Edition) Build #PY-221.5591.52 Colipot fails to update

### DISCUSSION THREAD

**Discussion by Bhartendu-Kumar** [community/community#17648] [Copilot Conversations] [answered]
I get this message when a notification for update comes and I click "Install"

The update of GitHub Copilot failed. Please try again later. Error message: Plugin 'GitHub Copilot' was not successfully downloaded

I can't see any update in Preferences->Plugins->Copilot

I am using a Macbook pro, M1 chip

**Comment by jansorg** [comment, marked as answer]
You can try to force an update with "Help > Check for updates". The latest version of the plugin (1.1.24.1610) contains a bug fix for the update notifications, which should hopefully fix your issue.

**Reply by Bhartendu-Kumar** [reply]
thanks for this , I could update to version : (1.1.24.1610). Lets see if this solves notification update failing issue.
